examples/websocket-relay/relay-app/src/lib/workflow.ts

```diff
@@ -233,226 +252,75 @@ function getTaskMessage(status: TaskStatus) {
   }
 }
 
-// JSON WORKFLOWS
-
-export const workflowOneJson = {
-  tasks: [
-    {
-      cause: null,
-      meta: {
-        memory: 4294967296,
-        time: 100000,
-      },
-      prf: [],
-      run: {
-        input: {
-          args: [
-            {
-              "/": "bafybeiejevluvtoevgk66plh5t6xiy3ikyuuxg3vgofuvpeckb6eadresm",
-            },
-            150,
-            350,
-            500,
-            500,
-          ],
-          func: "crop",
+// WORKFLOWS
+
+export const workflowOnePromised = WorkflowBuilder.workflow({
+  name: "one",
+  workflow: {
+    tasks: [
+      WorkflowBuilder.crop({
+        name: "crop",
+        resource: "bafybeiczefaiu7464ehupezpzulnti5jvcwnvdalqrdliugnnwcdz6ljia",
+        args: {
+          data: CID.parse(
+            "bafybeiejevluvtoevgk66plh5t6xiy3ikyuuxg3vgofuvpeckb6eadresm"
```

Comment:
```suggestion
          data: "{{ cid:bafybeiejevluvtoevgk66plh5t6xiy3ikyuuxg3vgofuvpeckb6eadresm }}"
```
actually you can do this and avoid `CID` dependency